sound/songs/se_291.s

```diff
@@ -1,11 +1,12 @@
     .include "asm/macros/c_decl.inc"
+	.include "asm/macros/portable.inc"
 	.include "sound/MPlayDef.s"
 
-	.section .rodata
+	SECTION_RODATA
 
 	.align 2
-	.global song0291_1
-song0291_1:
+	.global C_DECL(song0291_1)
+C_DECL(song0291_1):
```

Comment:
You only need to use C_DECL for things with ASM <-> C interop, not labels pointed to in assembly.

Reply:
Yep, this was debugging